### PR TITLE
[hmac,dv] update hmac unr

### PR DESCRIPTION
**Pull request by jdonjdon**
Regenerate full hmac unr (took 24hrs)

Signed-off-by: Jaedon Kim <jdonjdon@google.com>

### CODE REVIEW

**Comment by jdonjdon** [comment]
> That LGTM, but I think for UNR, we recommend to review with designer to make sure the excluded items make sense :) So I will leave it to Eli for the final approval.

ok @eunchan, can you review this when you have a chance?

File: hw/ip/hmac/dv/cov/hmac_cov_unr_excl.el [hunk 1]
```diff
@@ -1,18 +1,91 @@
...
+CHECKSUM: "780345221 4232397943"
+INSTANCE: tb.dut.u_sha2.u_pad
+ANNOTATION: "VC_COV_UNR"
+Block 17 "4027274935" "shaf_rdata = 32'b0;"
+ANNOTATION: "VC_COV_UNR"
+Block 21 "1204919414" "shaf_rdata = '0;"
+ANNOTATION: "VC_COV_UNR"
+Block 48 "1336235969" "st_d = StLenHi;"
+ANNOTATION: "VC_COV_UNR"
+Block 51 "1401847624" "st_d = StLenLo;"
```
Comment (eunchan) [inline hw/ip/hmac/dv/cov/hmac_cov_unr_excl.el:47]: I don't know if these statements can be waived or not.

**Review by eunchan — COMMENTED** [review]
You may waive the entire u_tlul_adapter instance, in my opinion.

I see other new coverage waivers in thes PR. It would be good if I can open verdi and see. Could you let me know how?

**Review by eunchan — COMMENTED** [review]
LGTM. Jaedon and I reviewed the items.

I delegate the approval to @cindychip

**Review by cindychip — APPROVED** [review]
Thanks! LGTM

**Pull request merged**